core/src/replay_stage.rs

```diff
@@ -1992,6 +1992,13 @@ impl ReplayStage {
         }
     }
 
+    /// Checks if it is time for us to start producing a leader block.
+    /// Additionally fails if:
```

Comment:
nit: `Additionally fails if:` -> `Fails if: Current Poh has not satisfied criteria to start my leader block`